packages/calcite-components/src/components/segmented-control-item/segmented-control-item.tsx

```diff
@@ -84,12 +98,9 @@ export class SegmentedControlItem {
       <Host aria-checked={toAriaBoolean(checked)} aria-label={value} role="radio">
         <label
           class={{
-            "label--scale-s": scale === "s",
-            "label--scale-m": scale === "m",
-            "label--scale-l": scale === "l",
-            "label--horizontal": layout === "horizontal",
-            "label--outline": appearance === "outline",
-            "label--outline-fill": appearance === "outline-fill",
+            [`label--scale-${scale}`]: true,
```

Comment:
Might be better to keep the previous class object since dynamically-generated classes won't work w/ our `CSS` lookup pattern (when this component is updated to use it).